packages/dds/tree/src/test/memory/tree.spec.ts

```diff
@@ -89,7 +98,8 @@ describe("SharedTree memory usage", () => {
 
 				public async run(): Promise<void> {
 					for (let i = 0; i < x; i++) {
-						this.sharedTree.root.propertyOne = x;
+						assert(this.sharedTree.root.child !== undefined);
```

Comment:
Hoist the assert to outside the for loop; doing it once should be enough for TS to stop complaining. Same below.